sunpy/visualization/animator/wcs.py

```diff
@@ -59,7 +60,12 @@ class ArrayAnimatorWCS(ArrayAnimator):
     ylabel: `string`, optional
        The yaxis label to use when drawing a line plot. Setting the label on
        the y-axis on an image plot should be done via ``coord_params``.
-
+    vmin: `int` or `str`, optional
+        Sets the minimum threshold for all slices of the animation. 
+        If specified as "auto", clips the minimum and maximum at each slider step to 1% and 99%
```

Comment:
Is this true anymore? Does it not just set the vmin at 1%?

Reply:
Oh yeah, vmin and vmax will have to be separated in the docstring